api/v1/types/model_types.go

```diff
@@ -31,3 +31,9 @@ type MetricResult struct {
 type MetricResultList struct {
 	Items []MetricResult `json:"items"`
 }
+
+type ExternalEntityListEntry struct {
```

Comment:
If this type is exposed over the wire, can we choose an alternative name? `Entity` is hard to interpret. How about `Point` or `MetricPoint` or something thats more descriptive?